apps/website/src/components/EditPageLink.astro

```diff
@@ -18,14 +20,31 @@ if (page === 'components') {
 const repoUrl = 'https://github.com/iTwin/iTwinUI';
 const githubEditUrl = `${repoUrl}/edit/main/apps/website/src/pages/docs/${fileName}`;
 ---
+<div>
 
-<a href={githubEditUrl} {...rest}>
-  <EditIcon aria-hidden='true' />Edit this page on GitHub
-</a>
+  <a href={githubEditUrl} {...rest}>
+    <EditIcon aria-hidden='true' />Edit
+  </a>
+
+  <a href='https://github.com/iTwin/iTwinUI/discussions/new?category=documentation' {...rest}>
```

Comment:
I looked into pre-populating the title and body of the discussion but was unable to find a solution.

Reply:
You can use `title` and `body` as query params

https://github.com/iTwin/iTwinUI/discussions/new?category=documentation&title=blah&body=blahblah

Reply:
Updated the title and body to include the path of the file feedback is being received from.